packages/react-native-editor/__device-tests__/pages/editor-page.js

```diff
@@ -24,7 +24,7 @@ class EditorPage {
 	accessibilityIdXPathAttrib;
 	paragraphBlockName = 'Paragraph';
 	verseBlockName = 'Verse';
-	orderedListButtonName = 'Convert to ordered list';
+	orderedListButtonName = 'Ordered';
```

Comment:
FYI, left a comment on the original PR about how I'm [a bit concerned that we search for single-work terms now](https://github.com/WordPress/gutenberg/pull/26135/files#r559567501). Nothing to change here but, let me know if you think it can be a problem as is @fluiddot , thoughts?

Reply:
Appium queries for elements by the accessibility label so it could be a problem if multiple elements have the same label. As long as we keep the label values unique it should be ok.

Reply:
OK, we'll have to be on the lookout 👍